OpenRA.Mods.Common/Activities/FindResources.cs

```diff
@@ -152,5 +152,16 @@ public override IEnumerable<Target> GetTargets(Actor self)
 		{
 			yield return Target.FromCell(self.World, self.Location);
 		}
+
+		public CPos GetFromSearchLocation(Actor self)
+		{
+			if (harv.LastLinkedProc != null)
+				return harv.LastLinkedProc.Location + harv.LastLinkedProc.Trait<IAcceptResources>().DeliveryOffset;
+			else if (harv.LinkedProc != null)
+				return harv.LinkedProc.Location + harv.LinkedProc.Trait<IAcceptResources>().DeliveryOffset;
+			else if (harv.LastOrderLocation != null)
```

Comment:
Why was this moved to lowest priority?  Won't this break all the cases where we *don't* want the closest ore?

Reply:
I agree. Using DeliveryOffset instead of Location should be the only change here.

LastOrderLocation needs to remain on top, else we'll definitely run into regressions.